apps/data/database/initdb.sql

```diff
@@ -39,28 +36,27 @@ CREATE TABLE listings (
     location LOCATION_TYPE NOT NULL,
     status STATUS_TYPE NOT NULL,
     description VARCHAR,
-    image_urls TEXT[],
+    image_urls TEXT [],
     created_at TIMESTAMP NOT NULL DEFAULT NOW(),
-    modified_at TIMESTAMP NOT NULL DEFAULT NOW()
+    modified_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    distance FLOAT DEFAULT 5.0
```

Comment:
why is distance being added? Distance should be calculated relative to where the user sends the request from, so it won't be a consistent value in the DB

Reply:
OK I see, I just noticed it in the GET listing API so I added it. so I need to calculate the distance between the user location and the listing location, and send it with the get request?

Reply:
Yes that's correct, there might be some library that does lat/long calculations to use but I imagine it wouldn't be too difficult to implement as our own util function either, up to you which route you wanna take

Reply:
OK I see, I just noticed it in the GET listing API so I added it. I'll remove that data attribute. Your right, it doesn't make much sense. Instead, would the logic be to take the user_id from the auth token - query and get user location. Get the listing location, then calculate the distance between them?

Reply:
Users don't have a location. do we update the user table to store a location, or do we get their location on front-end and pass it in either in the header or through the request body, maybe as "user_location". request body below:
![image](https://github.com/UVicMartletplace/martletplace/assets/65056763/ff45f809-cff4-4c61-b0d4-46ca48a374ab)

Reply:
user location will come in the request body